src/mpi/comm/comm_split_type.c

```diff
@@ -16,6 +16,11 @@
 #include "netloc_util.h"
 #endif
 
+#ifdef HAVE_NETLOC
+#include "netloc_util.h"
+#include <math.h>
```

Comment:
Where's `math.h` used?

Reply:
This one was added by mistake, I will remove this.